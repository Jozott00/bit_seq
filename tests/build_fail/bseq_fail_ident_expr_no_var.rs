use bit_seq::bseq;

fn main() {
    bseq!(non_exiting:1);
}