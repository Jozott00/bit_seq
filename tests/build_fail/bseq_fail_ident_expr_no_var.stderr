error[E0425]: cannot find value `non_exiting` in this scope
 --> tests/build_fail/bseq_fail_ident_expr_no_var.rs:4:11
  |
4 |     bseq!(non_exiting:1);
  |           ^^^^^^^^^^^ not found in this scope
